src/main/kotlin/com/chattriggers/ctjs/engine/Register.kt

```diff
@@ -23,11 +25,7 @@ object Register {
      * @param method The name of the method or the actual method to callback when the event is fired
      * @return The trigger for additional modification
      */
-    fun register(triggerType: Any, method: Any): Trigger {
-        require(triggerType is String) {
-            "register() expects a String as its first argument"
-        }
-
+    fun register(triggerType: String, method: Any): Trigger {
```

Comment:
This should definitely be in the API verification checker... is there a way to exclude an entire package except a certain class? The reverse definitely is.

Reply:
I dont think so, from what I've tried before (excluding utils package but including utils.vec). I'll see if I can do anything different when I try again tonight

Reply:
Well in the worst case we'll just have to change the package exclusion to exclude the sub packaged and tag the others with internal, won't be too terrible